Sources/ChatUI/ChatInChannel/MessageField/Giphy/View/GiphyAttributionMarkView.swift

```diff
@@ -0,0 +1,32 @@
+import SwiftUI
+
+class CustomTabBarController: UITabBarController {
+    var tabBarHeight: CGFloat {
+        tabBar.frame.size.height
+    }
+}
+
+struct GiphyAttributionMarkView: View {
+
+    private let PADDING_ABOVE_TAB = 2.5
+
+    @Environment(\.colorScheme) private var colorScheme
+
+    var body: some View {
+        VStack {
+            Spacer()
+            HStack {
+                Spacer()
+                Image(getImageName() , bundle: .module)
+                        .padding(.bottom, CustomTabBarController().tabBarHeight + PADDING_ABOVE_TAB)
+            }
+        }
+    }
+
+    private func getImageName() -> String {
```

Comment:
```suggestion
    private va imageName: String {
```
it doesn't need to use method. I recommend that you use a read-only property instead.